contentful_proxy/utils/cache/__init__.py

```diff
@@ -102,6 +102,7 @@ def _http_get(self, url, query):
             content_transformations = [
                 transformations.ReplaceAssetLinks(proxy_hostname=self.base_url),
                 transformations.ResolveIncludes(),
+                transformations.RemoveIncludes(),
```

Comment:
Can we enable it by default? 🤔We don't need `includes` when `ResolveIncludes` is enabled.

Reply:
True, I would enable it.